mintty/PKGBUILD

```diff
@@ -3,7 +3,7 @@
 pkgname=mintty
 pkgver=2.8.4
 pkgrel=1
-epoch=1
+epoch=2
```

Comment:
Git for Windows no longer modifies `mintty` Therefore this change should be unnecessary.